x-pack/plugins/security_solution/public/app/home/template_wrapper/index.tsx

```diff
@@ -73,11 +77,17 @@ export const SecuritySolutionTemplateWrapper: React.FC<SecuritySolutionPageWrapp
     const { show: isShowingTimelineOverlay } = useDeepEqualSelector((state) =>
       getTimelineShowStatus(state, TimelineId.active)
     );
+    const endpointMetadataIndex = useMemo<string[]>(() => {
+      return [ENDPOINT_METADATA_INDEX];
+    }, []);
+    const [, { indexExists: metadataIndexExists }] = useFetchIndex(endpointMetadataIndex, true);
+    const { indicesExist } = useSourcererScope();
+    const securityIndicesExist = indicesExist || metadataIndexExists;
 
     /* StyledKibanaPageTemplate is a styled EuiPageTemplate. Security solution currently passes the header and page content as the children of StyledKibanaPageTemplate, as opposed to using the pageHeader prop, which may account for any style discrepancies, such as the bottom border not extending the full width of the page, between EuiPageTemplate and the security solution pages.
```

Comment:
```suggestion
    // StyledKibanaPageTemplate is a styled EuiPageTemplate. Security solution currently passes the header and page content as the children of StyledKibanaPageTemplate, as opposed to using the pageHeader prop, which may account for any style discrepancies, such as the bottom border not extending the full width of the page, between EuiPageTemplate and the security solution pages.
```

Was this comment supposed to have line breaks and then something removed it? Because it looks really strange as it is with this line break. I would say just use `//` otherwise, do the line breaks?

Nit here, btw, but that looks strange nonetheless.

Reply:
I made the change